docs/examples/patterns/contextual-menu/contextual-menu.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: examples
-title: Contextual menu
+title: Contextual menu / base
```

Comment:
You don't need to repeat the pattern name in the filename. Keep it consistent with other examples. For example: `contextual-menu/default.html` and `contextual-menu/with-indicator.html`